jina/peapods/pods/__init__.py

```diff
@@ -331,6 +334,17 @@ async def scale(self, replicas: int):
                 )  # scale down has some challenges with the exit fifo
             self.pod_args.replicas = replicas
 
+        def has_forked_processes(self):
```

Comment:
make it a private property I'd say

Reply:
hmm why private? The Pod needs to use it